app/views/admin/statistics_announcements/show/_main.html.erb

```diff
@@ -0,0 +1,137 @@
+<div class="app-view-statistics-announcements-summary__main">
+  <section class="app-view-statistics-announcements-summary__section page-header">
```

Comment:
Are we using  the `page-header` class anywhere?

Reply:
Nup, can remove this. Although this change might get wrapped up in refactoring.